game-server/data/handlers/ai/instance/empyreanCrucible/RM1337AI.java

```diff
@@ -59,99 +59,64 @@ public void handleBackHome() {
 	@Override
 	public void handleAttack(Creature creature) {
 		super.handleAttack(creature);
-		if (isHome.compareAndSet(true, false)) {
+		if (isHome.compareAndSet(true, false))
 			startSkillTask1();
```

Comment:
This should be handled in `handleHpPhase` and `isHome` can be removed. You can add a 100% phase if required.